.claude/commands/pr.md

```
@@ -0,0 +1,139 @@
+# Smart PR Message Generator
+
+`/commit` 이후에 실행하여 Pull Request 메시지를 자동 생성하는 명령어입니다.
+
+## 사용법
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)